_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
+          <input type="text" name="veteran[mothers_maiden_name]"
```

Comment:
Oh, I think I misunderstood what we were trying to do. I thought we were creating components for each individual question. Now I understand why you were including the `id`s. Wish there was a better way of doing this, though, so our html is more semantic. We might be able to set a prop on `MothersMaidenName` when we attach the component to the form, like so: `<MothersMaidenName relation="veteran"/>`. And then instead of prefixing with unique ids we could prefix with the value of the relation prop?